catalog/raw/postgres/postgres-10.16-postgresql.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: 10.16-debian
 spec:
   coordinator:
-    image: kubedb/pg-coordinator:v0.2.0
+    image: kubedb/pg-coordinator:pg-upgrade-ops_linux_amd64
```

Comment:
Fix version?

Reply:
it will be v0.3.0. But this one will automatically update when we cut a new release.

Reply:
okay